plugin/trino-hive/src/main/java/io/trino/plugin/hive/HivePartitionedBucketFunction.java

```diff
@@ -1,108 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.hive;
-
-import io.trino.plugin.hive.util.HiveBucketing;
-import io.trino.plugin.hive.util.HiveBucketing.BucketingVersion;
-import io.trino.spi.Page;
-import io.trino.spi.block.Block;
-import io.trino.spi.connector.BucketFunction;
-import io.trino.spi.type.Type;
-import io.trino.spi.type.TypeOperators;
-import org.apache.hadoop.hive.serde2.typeinfo.TypeInfo;
-
-import java.lang.invoke.MethodHandle;
-import java.util.List;
-
-import static com.google.common.base.MoreObjects.toStringHelper;
-import static com.google.common.base.Throwables.throwIfUnchecked;
-import static com.google.common.collect.ImmutableList.toImmutableList;
-import static io.trino.spi.function.InvocationConvention.InvocationArgumentConvention.BLOCK_POSITION;
-import static io.trino.spi.function.InvocationConvention.InvocationReturnConvention.FAIL_ON_NULL;
-import static io.trino.spi.function.InvocationConvention.simpleConvention;
-import static java.util.Objects.requireNonNull;
-
-public class HivePartitionedBucketFunction
```

Comment:
where this one went?

Reply:
Sorry it was renamed but I screwed up the commit history, should be clearer now